autoload/vsnip/snippet.vim

```diff
@@ -66,11 +69,14 @@ function s:Snippet.init() abort
           let a:node.children = [vsnip#snippet#node#create_text(self.group[a:node.id].text())]
         endif
       else
+        let l:text = a:node.resolve({ 'prev_node': self.prev_node })
```

Comment:
It has a potential bug of the indentation fixing for the multiple lines.
We should provide `before_text` instead of `prev_node` for the `s:Variable.resolve`.

But it needs some of the refactor, so I leave it for now.